docs/tutorial/security.md

```diff
@@ -709,6 +709,97 @@ const { shell } = require('electron')
 shell.openExternal('https://example.com/index.html')
 ```
 
+## 15) Disable the `remote` module
+
+The `remote` module provides a simple way to do inter-process communication (IPC) between the renderer process (web page) and the main process. Using it, a renderer can invoke methods of a main process object without explicitly sending inter-process messages. If your desktop application does not run untrusted content, this is a very elegant way to have your renderer processes access and work with modules that are only available to the main process, such as GUI-related modules (dialog, menu etc.).
+
+However, if your app can run untrusted content and you sandbox your renderer processes accordingly, the `remote` module makes it easy for malicious code to escape the sandbox and have access to system resources via the higher privileges of the main process. Therefore, it should be disabled in such circumstances.
```

Comment:
**even if** you sandbox your renderer processes

Reply:
Excellent point, fixed.